src/Microsoft.Diagnostics.DebugServices.Implementation/ThreadService.cs

```diff
@@ -224,6 +229,43 @@ public bool TryGetRegisterValue(ReadOnlySpan<byte> context, int registerIndex, o
             return false;
         }
 
+        /// <summary>
+        /// Change a specific register. Currently only used to set the ContextFlags in certain implementations.
```

Comment:
Concern of the caller. We should not have the comment here.

Reply:
Comment removed.